website/source/docs/builders/amazon-ebs.html.md

```diff
@@ -173,6 +174,31 @@ builder.
 -   `skip_region_validation` (boolean) - Set to true if you want to skip 
     validation of the region configuration option.  Defaults to false.
 
+-   `source_ami_filter` (object) - Filters used to populate the `source_ami` field.
+    Example:
+    ``` {.javascript}
+    "source_ami_filter": {
+        "filters": {
+          "virtualization-type": "hvm",
+          "name": "*ubuntu-xenial-16.04-amd64-server-*",
+          "root-device-type": "ebs"
+        },
+        "owners": ["099720109477"],
+        "most_recent": true
+    }
+    ```
+    This selects the most recent Ubuntu 16.04 HVM EBS AMI from Canonical.
+
+-   `source_ami_filter.filters` (map of strings) - filters used to select a `source_ami`.
```

Comment:
This should be indented and only say `filters` as key, see `ami_block_device_mappings`. The same goes for `owners`, and `most_recent`.